workspace/spec/lita/handlers/teamcity_spec_anki.rb

```diff
@@ -425,7 +423,64 @@
       send_command("artifacts #{valid_build_name} #{valid_build_number}")
       expect(replies.last).to eq(expected_output)
     end
-
   end
 
+  describe '#running' do
+    running_build_json_str = '{"count": 2,
+                               "build": [{"buildTypeId": "CozmoOne_MasterBuild",
+                                          "branchName": "BI-123-XYZ",
+                                          "id": "4307",
+                                          "number":"1",
+                                          "percentageComplete": "1"},
+                                         {"buildTypeId": "OverDriveOne_Release_Build",
+                                          "branchName": "BI-124-XYZ",
+                                          "id": "4308",
+                                          "number": "2",
+                                          "percentageComplete": "2"}]}'
+    builds_json_str = '{"running-info": {"estimatedTotalSeconds": "10",
+                                         "elapsedSeconds": "1"}}'
+    time_diff = "09s"
+
+    before do
+      stub_request(:get, /running:true/).to_return(:status => 200, :body => running_build_json_str)
+      stub_request(:get, /id:/).to_return(:status => 200, :body => builds_json_str)
+    end
+
+    it 'replies all found running builds if wildcard = False' do
+      expected_output = "Here are the builds I found:\n\n<#{registry.config.handlers.teamcity.site}"\
+                        "/viewLog.html?buildId=4307"\
+                        "&buildTypeId=CozmoOne_MasterBuild|CozmoOne_MasterBuild> (1) - "\
+                        "`1\%` complete - Time left: `#{time_diff}` - BI-123-XYZ"\
+                        "\n<#{registry.config.handlers.teamcity.site}/viewLog.html?buildId=4308"\
+                        "&buildTypeId=OverDriveOne_Release_Build|OverDriveOne_Release_Build> (2) - "\
+                        "`2\%` complete - Time left: `#{time_diff}` - BI-124-XYZ"
+      send_command("running")
+      expect(replies.last).to eq(expected_output)
+    end
+
+    it 'replies the correct running builds if wildcard = True' do
+      build_id = "CozmoOne_MasterBuild"
+      expected_output = "Here are the builds I found:\n\n<#{registry.config.handlers.teamcity.site}"\
+                        "/viewLog.html?buildId=4307"\
+                        "&buildTypeId=#{build_id}|#{build_id}> (1) - "\
+                        "`1\%` complete - Time left: `#{time_diff}` - BI-123-XYZ"
+      send_command("running #{build_id}*")
+      expect(replies.last).to eq(expected_output)
+    end
+
+    it 'replies "error.request" if no connection to Teamcity' do
+      build_id = "CozmoOne_MasterBuild"
+      expected_output = "Error fetching TeamCity build types"
+      stub_request(:get, /running:true/).to_return(:status => 400, :body => [])
```

Comment:
If you cannot connect to Teamcity, it will not return empty list data. So you should use:
<code>stub_request(:get, /running:true/).to_raise(StandardError.new("connection error"))</code>